### DISCUSSION TITLE
jQueryUI 1.13.0 is not published on nuget.org

### DISCUSSION THREAD

**Discussion by auliere** [jquery/jquery-ui#2006] [Q&A] [answered]
Hello, jquery-ui community!

We consume jQuery-ui library through a following nuget package - [jQuery.UI.Combined](https://www.nuget.org/packages/jQuery.UI.Combined/) - on my project. However, this package has not been updated recently and it lacks the 1.13.0 version of the library.

We are forced to make an upgrade to 1.13.0 due to a recent security issue found in jquery-ui 1.12.1, however it is impossible to do that via a nuget package right now. 

Could you upload the most recent jquery-ui version as a nuget package, please, if this is possible at all?
Thank you.

**Comment by markvantilburg** [comment, marked as answer]
That package is maintained by outercurve. I have sent (them?) a message on nuget if they could update the package.

**Comment by auliere** [comment]
Thank you very much Mark! Please do send over the response here if you receive any.

**Comment by mgol** [comment]
Migrating to a discussion per @markvantilburg's comment.

**Comment by auliere** [comment]
The maintainer had published the new version of jquery ui combined package on nuget.org. Thank you guys for your help! I think this discussion can be closed.
![image](https://user-images.githubusercontent.com/12403926/142194115-bdd39984-c7dd-4ed8-a663-3ce3b8e8ee09.png)
